Sources/Sentry/SentryClient.m

```diff
@@ -229,8 +229,9 @@ - (void)captureEnvelope:(SentryEnvelope *)envelope
 {
     // TODO: What is about beforeSend
 
-    if (nil == self.options.parsedDsn) {
-        [SentryLog logWithMessage:@"No DSN set. Won't do anyting." andLevel:kSentryLogLevelDebug];
+    if (!self.options.enabled || nil == self.options.parsedDsn) {
```

Comment:
M: can we extract this `if` check and the one in `captureUserFeedback` to a method that returns a bool so we reuse the method?

Reply:
That's a good point.